releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -6,6 +6,14 @@
 
 [Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
 
+## 2.3.0 - January 05, 2022
+
+* Fixed a bug where `spring.cloud.application` was still used in a location. This caused a refresh bug where a null value was used on refresh. `spring.cloud.application` has replaced by `key-filter`.
```

Comment:
The last sentence is not clear to me. Which has replaced which? Either 
`spring.cloud.application` has now replaced `key-filter`
or 
`spring.cloud.application` has now been replaced by `key-filter`